src/uniprotkb/components/entry/__tests__/__mocks__/entryPublicationsData.json

```diff
@@ -1,157 +1,1824 @@
 {
-  "facets": [{
-    "label": "Study type",
-    "name": "study_type",
-    "allowMultipleSelection": false,
-    "values": [{
-      "label": "Small scale",
-      "value": "small_scale",
-      "count": 239
-    }, {
-      "label": "Large scale",
-      "value": "large_scale",
-      "count": 8
-    }]
-  }],
-  "results": [{
-      "reference": {
-        "citation": {
-          "citationType": "UniProt indexed literatures",
-          "authors": [
-            "Kang J.",
-            "Lemaire H.-G.",
-            "Unterbeck A.",
-            "Salbaum J.M.",
-            "Masters C.L.",
-            "Grzeschik K.-H.",
-            "Multhaup G.",
-            "Beyreuther K.",
-            "Mueller-Hill B."
-          ],
-          "citationCrossReferences": [{
-              "database": "PubMed",
-              "id": "2881207"
+  "facets": [
```

Comment:
This needs to be transformed to a .ts file, please comment the URL that was used to get the data in order to be able to re-fetch it later in case of another schema change